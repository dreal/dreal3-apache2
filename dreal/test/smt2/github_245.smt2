(set-logic QF_NRA)
(declare-fun a () Real)
(declare-fun b () Real)
(define-fun get_R ((x Real) (y Real)) Real
(* x (cos y)))
(assert (= a 12))
(assert (= b -0.7853981633974483))
(declare-fun R () Real)
;(assert (= R (* a (cos b))))
(assert (= R (get_R a b)))
(check-sat)
(get-model)

