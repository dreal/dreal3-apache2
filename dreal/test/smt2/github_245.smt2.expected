delta-sat with delta = 0.001
(model
  (define-fun a () Real 12)
  (define-fun b () Real [-0.78539816339744839, -0.78539816339744828])
  (define-fun R () Real [8.4852813742385571, 8.4852813742385855])
)
